workshops/ajar/README.md

```diff
@@ -21,6 +57,14 @@ been written):
 
 __More coming soon...__
 
+## Tips And Tricks
+
+1. To open a link in another tab, right click on the link and select `Open In
```

Comment:
It's not really clear what links you are referring to.

Something along the lines of "We link to other websites in this workshop. So that you never have to leave this page, open links in new tabs by doing..."